grafana/resource_builtin_role_assignment_test.go

```diff
@@ -48,6 +48,147 @@ func TestAccBuiltInRoleAssignment(t *testing.T) {
 	})
 }
 
+func TestAccBuiltInRoleAssignmentUpdate(t *testing.T) {
+	CheckEnterpriseTestsEnabled(t)
+
+	var br gapi.BuiltInRoleAssignment
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: testAccProviderFactories,
+		CheckDestroy:      testAccBuiltInRoleAssignmentCheckDestroy(&br),
```

Comment:
Should the `CheckDestroy` function also check that the default assignements weren't destroyed?

Reply:
Really good catch. I actually just realized that `testAccBuiltInRoleAssignmentCheckDestroy` checks nothing, as the pointer to `&br` never updates to the created built-in role assignment, so we always pass an empty struct to the method, which returns success on it's turn as there is indeed no assignment for empty built-in role. 

Working on a fix.

Reply:
5566c8b4691e45c25d4b1383e155229156eda19f